src/types/server/api/login/handlers.d.ts

```diff
@@ -0,0 +1,9 @@
+export type EmailProperty = {
+  email: string
+}
+
+type OtpProperty = {
+  otp: string
+}
+
+export type VerifyOtpRequest = EmailProperty & OtpProperty
```

Comment:
I've recently read about the use-case of `.d.ts` files  to provide typescript with type information for a [library written in javascript](https://stackoverflow.com/questions/21247278/about-d-ts-in-typescript). Based on that comment it didnt feel right to use them in our pure-typescript server-side codebase and have been discouraging Jason from using it. Should we stick with the `.ts` extension for this file?

Reply:
Fair point, we can migrate them in another PR - will merge first to save the need for another approval.